src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -484,8 +473,7 @@ static void rr_handover(grpc_exec_ctx *exec_ctx, glb_lb_policy *glb_policy,
     grpc_lb_policy_pick(exec_ctx, glb_policy->rr_policy, &pp->pick_args,
```

Comment:
You are right. I guess it's unlikely that the RR policy, very recently created, would be ready to return anything immediately, which is why this hasn't been flushed out by tests.

Given that this is the second place where we pick from RR and have to branch on the result, I've factored out the logic into `pick_from_internal_rr()`.